biblemulticonverter/src/main/java/biblemulticonverter/format/paratext/AbstractParatextFormat.java

```diff
@@ -47,6 +35,21 @@
 import biblemulticonverter.format.paratext.ParatextCharacterContent.VerseStart;
 import biblemulticonverter.format.paratext.model.ChapterIdentifier;
 import biblemulticonverter.format.paratext.model.VerseIdentifier;
+import biblemulticonverter.format.paratext.utilities.LocationParser;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.EnumMap;
+import java.util.EnumSet;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
```

Comment:
This seems to be another case where Eclipse and IDEA auto-format differently:

![image](https://user-images.githubusercontent.com/1568931/101548796-cbb92a00-39ac-11eb-9a79-1bcc4cf29e7a.png)

(These are default settings)